config/config.go

```diff
@@ -30,6 +31,12 @@ type QueryTweak struct {
 	AlignTimestampsToStep  bool              `yaml:"align_timestamps_to_step" json:"alignTimestampsToStep,omitempty"`
 	DropResultLabels       []model.LabelName `yaml:"drop_result_labels" json:"dropResultLabels,omitempty"`
 	IgnoreFirstStep        bool              `yaml:"ignore_first_step" json:"ignoreFirstStep,omitempty"`
+	ApplyQuerySubstitution Substitution      `yaml:"apply_query_substitution" json:"applyQuerySubstitution,omitempty"`
```

Comment:
How about just making this a map (old -> new)? That would then also allow doing multiple substitutions.

How about `ReplaceStringInQuery` or so for an even clearer name of what this option does?

Reply:
Yes, I will do that (probably next week). Thanks!